lib/auth/bot_test.go

```diff
@@ -67,7 +200,7 @@ func TestBotOnboardFeatureDisabled(t *testing.T) {
 	_, err := createBotRole(context.Background(), srv.Auth(), "test", "bot-test", []string{})
 	require.NoError(t, err)
 
-	_, err = createBotUser(context.Background(), srv.Auth(), botName, botResourceName)
+	_, err = createBotUser(context.Background(), srv.Auth(), botName, botResourceName, wrappers.Traits{})
```

Comment:
bit: You should probably just create one context, store it as a variable and pass it to every function instead of calling `context.Background()` for every call.

Reply:
Sure, I'm all for switching tests to have a single `context.Background()` where possible. I think I was just modifying this line so it compiled, but I'm happy to change this as well.

Reply:
https://github.com/gravitational/teleport/pull/15117/commits/3064ed7c6b9558fe8d43c53900bd62582f610ca4